randovania/interface_common/persisted_options.py

```diff
@@ -154,6 +155,23 @@ def _msr_enable_remote_lua(options: dict) -> None:
         options["game_samus_returns"]["cosmetic_patches"].pop("enable_remote_lua", None)
 
 
+def _msr_citra_to_azahar(options: dict) -> None:
+    if "target_platform" in options.get("game_samus_returns", {}):
+        if options["game_samus_returns"]["target_platform"] == "citra":
+            options["game_samus_returns"]["target_platform"] = "azahar"
+
+    if "output_preference" in options.get("game_samus_returns", {}):
+        as_dict: dict = json.loads(options["game_samus_returns"]["output_preference"])
+        if as_dict.get("selected_tab", "") == "citra":
+            as_dict["selected_tab"] = "azahar"
+        # dirty replacement: replaces the platform and the field name for the always empty dict tab options
+        tab_options = as_dict.get("tab_options", None)
+        if tab_options:
+            tab_options["azahar"] = {}
+            tab_options.pop("citra", None)
+        options["game_samus_returns"]["output_preference"] = json_lib.dumps_small(as_dict)
```

Comment:
we shouldn't rely on RDV functions here. probably better/safer to replace the json_lib call with 
`json.dumps(obj, separators=(",", ":"))`

Reply:
Can you explain the difference since I didn't write this?

Reply:
Doesn't really answer my question but ok.